src/methods/staking/validate.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Args,
+  BaseTxInfo,
+  createMethod,
+  UnsignedTransaction
+} from '../../util';
+
+export interface StakingValidateArgs extends Args {
+  /**
+   * Declare the desire to nominate `targets` for the origin controller.
```

Comment:
In the rust code, this is a `Perbill`. Does submitting e.g. 5 here imply a 5% commission?
```suggestion
   * Set the desired commission for the validator. Value is {percent, Perbill, ???}.
```

Reply:
No it's perbill here too.